tests/phpunit/tests/functions.php

```diff
@@ -1096,6 +1106,21 @@ public function test_wp_extract_urls() {
 		$this->assertSame( array_slice( $original_urls, 0, 8 ), $urls );
 	}
 
+	/**
+	 * @ticket 30580
+	 *
+	 * Tests for backward compatibility of `wp_extract_urls` to remove unused semicolons.
```

Comment:
I would suggest putting the description first, followed by the ticket, and adding an `@covers` annotation to improve the accuracy of coverage reports. Like so:

```php
	/**
	 * Tests for backward compatibility of `wp_extract_urls()` to remove unused semicolons.
	 *
	 * @ticket 30580
	 *
	 * @covers ::wp_extract_urls
	 */
```

See `test_add_query_arg_removes_question_mark()` in the same file which uses the same format.